packages/expo/src/Linking/Linking.ts

```diff
@@ -136,6 +136,13 @@ function parse(url: string): ParsedURL {
 
 async function parseInitialURLAsync(): Promise<ParsedURL> {
   const initialUrl = await Linking.getInitialURL();
+  if (!initialUrl) {
```

Comment:
so this is an unrelated change but the expo package build step errored without it, because `parse` param must be a string. @EvanBacon / @ide please fix if wrong